NiL.JS/BaseLibrary/Error.cs

```diff
@@ -66,13 +66,54 @@ public Error(string message)
         private void makeCallStack()
         {
             StringBuilder res = new StringBuilder();
-            var context = Context.CurrentContext;
-            while (context != null)
+            var currentContext = Context.CurrentContext;
+            var contexts = Context.GetCurrectContextStack();
+            res.Append("    at ").AppendLine(currentContext.CodeNode?.ToString() ?? string.Empty);
+            for (int i = contexts.Count - 1; i > 0; i--)
             {
-                res.Append("in ").AppendLine(context.caller == null ? "" : (context.caller.name ?? "<anonymous method>"));
-                context = context.oldContext;
+                var context = contexts[i];
+                if (context != null)
+                {
+                    var line = FromTextPosition(context.RootContext.Code, context.CodeNode.Position);
+                    res.Append("    at ").AppendLine($"{context._owner?.name ?? "<anonymous method>"}({context._module?.FilePath ?? "<anonymous>"}:{line})");
+                }
             }
-            callstack = res.ToString();
+            callstack = Context.CurrentGlobalContext.ProxyValue(res.ToString());
+        }
+
+        public static int FromTextPosition(string text, int position)
```

Comment:
Is it copy-paste from `CodeCoordinates.FromTextPosition`? Why not use this function directly?

Reply:
because I want to return only line
but now  I have a bigger problem, in codeBlock.evaluateLines, I thought it would go into every line of code, but i was wrong, so I choose to close this pr now
do you have time to implement of callstack
plz

Reply:
I tried this, but did not get success. So, maybe i will try it yet once